### PR TITLE
Newprayer jk uub o etiop 3

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Created new work file based on https://github.com/BetaMasaheft/Documentation/issues/2572#issuecomment-2217327729

### CODE REVIEW

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/LIT7026PrayerGod.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7026PrayerGod" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Prayer to God the Father</title>
```
Comment (eu-genia) [inline new/LIT7026PrayerGod.xml:8]: `en` (the language of the title used)

Comment (eu-genia) [reply new/LIT7026PrayerGod.xml:8]: (Or t1 gez ኦእግዚኦ፡ ነአኵተከ፡ ወንሴብሐከ፡ ወንሰግድ፡ ለዕበየ፡...)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT7026PrayerGod.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT7026PrayerGod.xml:8]: I think this is not updated

Comment (karljonaskarlsson) [reply new/LIT7026PrayerGod.xml:8]: Thank you!

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**